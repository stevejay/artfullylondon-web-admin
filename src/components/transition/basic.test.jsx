import React from 'react'
import { mount } from 'enzyme'

import BasicTransition from '_src/components/transition/basic'

it('should render correctly', () => {
  const wrapper = mount(<BasicTransition><div id='child' /></BasicTransition>)
  expect(wrapper).toMatchSnapshot()
})
