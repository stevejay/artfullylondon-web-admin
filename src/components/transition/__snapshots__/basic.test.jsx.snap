// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<BasicTransition
  appear={false}
  className=""
  in={false}
  mountOnEnter={false}
  timeoutMs={200}
  unmountOnExit={false}
>
  <Transition
    appear={false}
    enter={true}
    exit={true}
    in={false}
    mountOnEnter={false}
    onEnter={[Function]}
    onEntered={[Function]}
    onEntering={[Function]}
    onExit={[Function]}
    onExited={[Function]}
    onExiting={[Function]}
    timeout={200}
    unmountOnExit={false}
  >
    <div
      className="  out"
      id="child"
    />
  </Transition>
</BasicTransition>
`;
